pages/common/xml-canonic.md

```diff
@@ -0,0 +1,20 @@
+# xml canonic
+
+> Make XML documents canonical.
+> More information: <http://xmlstar.sourceforge.net/docs.php>.
+
+- Make an XML document canonical, preserving comments (default behavior):
+
+`xml canonic --with-comments {{path/to/input.xml|URI}} >{{path/to/output.xml}}`
```

Comment:
If that's the default behaviour, then why not just remove the `--with-comments` flag? I think if we would include it the example should be something like `Make an XML document canonical, explicitly preserving comments`.